packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/chat.tests.js

```diff
@@ -84,6 +85,7 @@ const moduleConfig = {
         this.getMessageGroups = () => this.$element.find(`.${CHAT_MESSAGEGROUP_CLASS}`);
         this.getDayHeaders = () => this.$element.find(`.${CHAT_MESSAGELIST_DAY_HEADER_CLASS}`);
         this.getBubbles = () => this.$element.find(`.${CHAT_MESSAGEBUBBLE_CLASS}`);
+        this.getBubblesContent = () => this.$element.find(`.${CHAT_MESSAGEBUBBLE_CONTENT_CLASS}`);
```

Comment:
Do you mean `getBubbleContents`?

Reply:
https://github.com/DevExpress/DevExtreme/pull/28358/commits/44d392b5a8aa09163beda3e50ea34fad7db3a4f0